src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -947,27 +1042,44 @@ private bool TryProcessEventCore(CompilationEvent e, AnalysisScope analysisScope
         /// True, if successfully executed the actions for the given analysis scope OR no actions were required to be executed for the given analysis scope.
         /// False, otherwise.
         /// </returns>
```

Comment:
Please update doc comment to match the revised contract.

Reply:
Done